namespaces/live-1.cloud-platform.service.justice.gov.uk/contact-moj-development/resources/rds.tf

```diff
@@ -0,0 +1,55 @@
+################################################################################
+# contact-moj
+# Application RDS (PostgreSQL)
+#################################################################################
+
+module "contact-moj_rds" {
+  source                     = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.1"
+  cluster_name               = var.cluster_name
+  cluster_state_bucket       = var.cluster_state_bucket
+  team_name                  = "correspondence"
+  business-unit              = "Central Digital"
+  application                = "contact-moj"
+  is-production              = "false"
+  db_engine                  = "postgres"
+  db_engine_version          = "9.5"
+  db_backup_retention_period = "7"
+  db_name                    = "contact-moj_development"
```

Comment:
@digitalali-moj ,  db_name can only contain letters and underscores. Can you amend please.

Reply:
OK thanks, done